kernel/indtypes.ml

```diff
@@ -49,14 +46,11 @@ let weaker_noccur_between env x nvars t =
    if noccur_between x nvars t' then Some t'
    else None
 
-let is_constructor_head t =
-  isRel(fst(decompose_app t))
-
 (************************************************************************)
 (* Various well-formedness check for inductive declarations            *)
 
 (* Errors related to inductive constructions *)
-type inductive_error =
+type inductive_error = IndTyping.inductive_error =
```

Comment:
Why the duplication? Should it be deprecated?

Reply:
>Should it be deprecated?

Maybe. I may try moving it to typing_errors at the same time.